src/lucky_cli/process_runner.cr

```diff
@@ -42,6 +42,24 @@ module LuckyCli
     end
   end
 
+  class ProcessRunner::Nox < ProcessRunner
```

Comment:
For now I did the minimal change just to add in Nox to the mix. I will follow up this PR with another that rips out all this ProcessRunner stuff since we will no longer need it. 

The resulting code will literally just be:

```crystal
class LuckyCli::Dev < LuckyTask::Task
  summary "Start your app with a process runner and Procfile.dev"

  def call
    Nox.run("Procfile.dev")
  end
end
```